basics/demo/lib/posts.js

```diff
@@ -9,12 +9,12 @@ const postsDirectory = path.join(process.cwd(), 'posts');
 export function getSortedPostsData() {
   // Get file names under /posts
   const fileNames = fs.readdirSync(postsDirectory);
-  const allPostsData = fileNames.map((fileName) => {
+  const allPostsData = fileNames.map((fileNames) => {
```

Comment:
When iterating over the map, the correct variable name here would be `fileName`. This matches what is live today.